nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -339,6 +339,19 @@ def _impl(inputs, attr, params):
             extras={'axis': axis.asnumpy()[0]})(inputs, attr)
     return _impl
 
+def _pack():
+    def _impl(inputs, attr, params):
+        shape0 = attr['_input_shapes'][inputs[0]][0].copy()
+        shape = shape0.copy()
+        axis = int(attr["axis"])
+        if axis < 0:
```

Comment:
Can you avoid this check? Topi implementation of concatenate op has provision to handle negative axis.

Reply:
I advice not to relay on ```_input_shapes``` for transforms as there may be empty dimensions or -1 possible which might fail later.

Reply:
I think it will be better if we can implement a operator using existing operators, unless there is such requirement.

Reply:
@grwlf you may try using expand_dim instead of inserting '1' in ```_input_shapes```.

Reply:
Thank you, I'll try to use expand_dim